### ISSUE TITLE
Reproducibility

### ISSUE THREAD

**Issue by yl2428** [ASK-Berkeley/EScAIP#11]
Hello!

1. We attempted to run the SPICE dataset, and following the references in your paper, we downloaded the data from the MACE-off233 repository at [https://www.repository.cam.ac.uk/items/d50227cd-194f-4ba4-aeb7-2643a69f025f](https://www.repository.cam.ac.uk/items/d50227cd-194f-4ba4-aeb7-2643a69f025f). However, we found that ESCAIP does not support the XYZ dataset, so we were unable to train the SPICE dataset.

2. The SPICE dataset includes several subsets for evaluation, such as PubChem, DeShaw, etc., but we could not find the specific evaluation code.

3. We tried to reproduce the ESCAIP results on OC20 2M. Using the current configuration, we attempted to replicate your results on a 4-GPU machine, but the outcomes were not very promising. The YAML file you provided specifies parameters like batch size = 12, epochs = 30, and learning rate = 0.0001, but it does not indicate how many GPUs were used for training. If possible, providing a snippet of the initial loss reduction trajectory would greatly help in our reproduction efforts.

4. For OC22, we encountered similar issues as with OC20. How many GPUs were used for training in this case? We also couldn’t find the YAML file used for ESCAIP on OC22. Could it be that this file was accidentally omitted from the submission?

5. Additionally, we noticed that the MD22 YAML file is not runnable, it has some problems with the pbc.  Could you clarify if there are specific configurations or updates needed for this scenario?

Thank you so much.

**Comment by EricZQu** [comment]
Hey, thanks for trying out and testing our model, for your questions:
1. We use this [script](https://github.com/FAIR-Chem/fairchem/blob/4f42b677006a2f7c2916dcf531d2fdad19b8ba54/src/fairchem/core/preprocessing/atoms_to_graphs.py) from FAIRChem to convert the XYZ dataset into the pyg Data format, and then write it into lmdb format. An example on writing to lmdb format can be found in this [script](https://github.com/ASK-Berkeley/EScAIP/blob/main/scripts/convert_lmdb_md22.py)
2. They are evaluated using a jupiter notebook in our dev repo, we haven't released it because it was not cleaned up. Let us know if you would like it to be released.
3. The hyperparameters included in our config file is tuned for training on 128 GPUs. Here is a screenshot of the entire training trajectory:
![Image](https://github.com/user-attachments/assets/95c90a24-b8d2-4bcd-aa89-05443c1e4451)
4. Yes it was accidentally omitted. Thank you for pointing it out. We will upload it in a release soon.
5. Can you provide more details on this? I haven't encountered it during my tests.
